NitroxClient/Debuggers/EntityDebugger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures;
+using UnityEngine;
+
+namespace NitroxClient.Debuggers
+{
+    public class EntityDebugger : BaseDebugger
+    {
+        public EntityDebugger() : base(200, null, KeyCode.E, true, false, false, GUISkinCreationOptions.DERIVEDCOPY)
+        {
+            ActiveTab = AddTab("EntityDebugger", RenderEntityDebugger);
+        }
+
+        private void RenderEntityDebugger()
+        {
+        }
+
+        public override void OnGUIExtra()
+        {
+            foreach (KeyValuePair<NitroxId, GameObject> gameObjectPairs in NitroxEntity.GetGameObjects())
+            {
+                NitroxId id = gameObjectPairs.Key;
+                GameObject gameObject = gameObjectPairs.Value;
```

Comment:
If it's just a reference I think it's better to use `GameObject gameObject => gameObjectPairs.Value;`